packages/imperative/src/utilities/src/TextUtils.ts

```diff
@@ -39,7 +39,7 @@ export class TextUtils {
     public static getRecommendedWidth(preferredWidth: number = TextUtils.DEFAULT_WRAP_WIDTH): number {
         const widthSafeGuard = 8; // prevent partial words from continuing over lines
         const yargs = require("yargs");
-        const maxWidth = !isNullOrUndefined(yargs.terminalWidth() && yargs.terminalWidth() > 0) ?
+        const maxWidth = !((yargs.terminalWidth() && yargs.terminalWidth() > 0) == null) ?
```

Comment:
This accurately reflects the old logic, but I think the old logic may have been wrong because the boolean statement won't evaluate to `null`. Should we change it to the following instead to match the vNext branch?
```suggestion
        const maxWidth = (!(yargs.terminalWidth() == null) && yargs.terminalWidth() > 0) ?
```